src/test/java/seedu/address/model/pool/PooledPassengerContainsKeywordsPredicateTest.java

```diff
@@ -50,7 +50,7 @@ public void test_nameContainsKeywords_returnsTrue() {
         // One keyword
         PooledPassengerContainsKeywordsPredicate predicate =
                 new PooledPassengerContainsKeywordsPredicate(
-                        Arrays.asList(ALICE.getName().toString().split("\\s")[0]));
+                        Collections.singletonList(ALICE.getName().toString().split("\\s")[0]));
```

Comment:
@chewterence what is your opinion on using the testUtil's passengers constant and extract a single keyword from the name like this? (Since the name constants are always in two words but we need to test different word length with the predicate)

Reply:
@JoelHo might be able to give a better insight to this